streams-tests/shared/src/test/scala/zio/stream/experimental/ZStreamSpec.scala

```diff
@@ -44,6 +44,68 @@ object ZStreamSpec extends ZIOBaseSpec {
           } yield assert(res1)(fails(equalTo(s))) && assert(res2)(fails(equalTo(s)))
         })
       ),
+      suite("aggregate")(
+        testM("aggregate") {
+          val s      = ZStream('1', '2', ',', '3', '4')
+          val parser = ZTransducer.collectAllWhile[Char](_.isDigit).map(_.mkString.toInt)
```

Comment:
The current version of the transducer for this test is:
```
val parser = ZSink.collectAllWhile[Char](_.isDigit).map(_.mkString.toInt) <* ZSink
          .collectAllWhile[Char](_ == ',')
```
Nice simplification. (Also, ZTransducers may not form monads after all)